src/common/metric.tsx

```diff
@@ -95,17 +95,26 @@ export function getMetricDisclaimer(metric: Metric) {
   return METRIC_TO_DISCLAIMER[metric];
 }
 
-export function getMetricStatusText(metric: Metric, projection: Projection) {
-  const METRIC_TO_STATUS_TEXT: { [metricName: number]: string } = {
-    [Metric.CASE_GROWTH_RATE]: CaseGrowth.caseGrowthStatusText(projection),
-    [Metric.POSITIVE_TESTS]: TestRates.positiveTestsStatusText(projection),
-    [Metric.HOSPITAL_USAGE]: Hospitalizations.hospitalOccupancyStatusText(
-      projection,
-    ),
-    [Metric.CONTACT_TRACING]: ContactTracing.contactTracingStatusText(
-      projection,
-    ),
-    [Metric.CASE_DENSITY]: CaseDensity.caseDensityStatusText(projection),
-  };
-  return METRIC_TO_STATUS_TEXT[metric];
+// CASE_GROWTH_RATE,
+// POSITIVE_TESTS,
+// HOSPITAL_USAGE,
+// CONTACT_TRACING,
+// FUTURE_PROJECTIONS,
+// CASE_DENSITY,
+const metricDefinitions: { [metric in Metric]: MetricDefinition } = {
+  [Metric.CASE_GROWTH_RATE]: CaseGrowth.metricCaseGrowth,
+  [Metric.POSITIVE_TESTS]: TestRates.metricPositiveTestRate,
+  [Metric.HOSPITAL_USAGE]: Hospitalizations.metricICUHeadroom,
+  [Metric.CONTACT_TRACING]: ContactTracing.metricContactTracing,
+  [Metric.FUTURE_PROJECTIONS]: FutureProjections.metricFutureProjection,
+  [Metric.CASE_DENSITY]: CaseDensity.metricCaseIncidence,
+};
+
+export function getMetricStatusText(metric: Metric, projections: Projections) {
+  if (!(metric in metricDefinitions)) {
+    fail('unknown metric');
+  }
```

Comment:
Do we even need this check? 🤔

Reply:
_actually_, Future Projections is not a core indicator, it doesn't share all the UI elements that the indicators have, so I might have to keep that distinction here.